website/docs/r/servicebus_subscription.html.markdown

```diff
@@ -92,7 +92,7 @@ The following arguments are supported:
 
 ### TimeSpan Format
 
-Some arguments for this resource are required in the TimeSpan format which is used to represent a length of time. The supported format is documented [here](https://msdn.microsoft.com/en-us/library/se73z7b9(v=vs.110).aspx#Anchor_2)
```

Comment:
Just remove this "TimeSpan Format" as ISO8601 is a well-known format.
(PS: we normally add link to azure rest api instead of azure cli/azure pwsh as a reference, the link for this case is [this](https://docs.microsoft.com/en-us/rest/api/servicebus/subscriptions/createorupdate#request-body))

Reply:
I think its nice to have the ISO8601 format link, as we can over time add more examples. Also, it allows us to link to the actual documentation. Hope you keep it.

Reply:
Of course, then we can follow the way as other resource did (e.g. [iothub](https://github.com/terraform-providers/terraform-provider-azurerm/blob/master/website/docs/r/iothub.html.markdown)), add the link towards wikipedia for ISO8601. The point is that we'd better keep things consistent.

Reply:
I'm inclined to agree, it's easy enough to link to public examples like iothub for thsoe who need the examples while keeping the docs clean,

Reply:
So take out link and examples?

Reply:
Made the documentation consistent with Topic and Queue documentation. Also, please note, that it looks like your provider does not support duplicateDetectionHistoryTimeWindow (https://docs.microsoft.com/en-us/rest/api/servicebus/subscriptions/createorupdate). Let me know and I will add a separate issue for that.